x/concentrated-liquidity/internal/emit.go

```diff
@@ -0,0 +1,71 @@
+package events
+
+import (
+	"strconv"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/osmosis-labs/osmosis/v13/x/gamm/types"
+)
+
+// TODO: change as needed for CL rather than gamm
+func EmitSwapEvent(ctx sdk.Context, sender sdk.AccAddress, poolId uint64, input sdk.Coin, output sdk.Coin) {
```

Comment:
Can we instead move the swap event logic to swap router, make it public and import it to gamm and here?